debian/patches/series

```diff
@@ -0,0 +1,2 @@
+# static-bfd.patch
+undo-upstream.patch
```

Comment:
I wonder if this patch really does get applied.  Our debian builds actually take their patches from the specfile.  I'm not sure if we will apply patches from this dir if they exist.  I will check the build.

So, yeah, this is the problem.  If you add this patch to the RPM specfile, things should work better.

I suppose we really ought to handle this situation better.